web/src/components/RunsTable/runsTable.js

```diff
@@ -159,6 +159,16 @@ class RunsTable extends Component {
             }
           }
 
+          // If a comment exist, add it to notes field.
+          if ('meta' in data) {
+            const meta = data['meta']
+            var comment = ''
+            if ('comment' in meta) {
+              comment = meta['comment']
+            }
+            data = {...data, 'notes': comment}
```

Comment:
```suggestion
            
```

Reply:
If we add `notes` to data in this block, it will get overridden while expanding `omniboard` columns in the next block. So, my suggestion is to add notes to `data['omniboard']`.